AEM/optimization-for-tensorflow-installation-guide.md

```diff
@@ -161,23 +161,42 @@ There are multiple options provided to download Intel® AI Analytics Toolkit, in
 
 Available for Linux\* and Windows\*
 
-TensorFlow version: 2.11.0
+TensorFlow version >= 2.12.0
```

Comment:
why do we add new section for 2.12? Can we merge it to 2.10 to 2.11 section?

Reply:
because supported python versions are different